pathspider/plugins/dscp.py

```diff
@@ -18,32 +18,37 @@ class DSCP(SynchronizedSpider, PluggableSpider):
     chains = [BasicChain, DSCPChain, TCPChain, DNSChain]
     connect_supported = ["http", "tcp", "dnstcp", "dnsudp"]
 
-    def config_no_dscp(self):  # pylint: disable=no-self-use
-        """
-        Disables DSCP marking via iptables.
-        """
-
-        logger = logging.getLogger('dscp')
-        for iptables in ['iptables', 'ip6tables']:
-            subprocess.check_call([iptables, '-t', 'mangle', '-F'])
-        logger.debug("Configurator disabled DSCP marking")
-
-    def config_dscp(self):
-        """
-        Enables DSCP marking via iptables.
-        """
+    def generate_f(self, i):
         logger = logging.getLogger('dscp')
```

Comment:
Can this be removed? We don't actually use it, and then we get the logger within the function anyway.